manifests/s/Smartedu/Smartedu/1.0.0/Smartedu.Smartedu.locale.en-US.yaml

```diff
@@ -0,0 +1,19 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Smartedu.Smartedu
+PackageVersion: 1.0.0
+PackageLocale: en-US
+Publisher: Ministry of Education of the People's Republic of China
+PublisherUrl: https://basic.smartedu.cn/
+PublisherSupportUrl: https://basic.smartedu.cn/
+PrivacyUrl: https://basic.smartedu.cn/
```

Comment:
```suggestion
PublisherUrl: https://www.smartedu.cn/
PublisherSupportUrl: https://www.smartedu.cn/special/helpcenter
# PrivacyUrl:
```

These three URLS should not be the same as the PackageURL. They should link to the website of the publisher, the support page or contact us form, and the privacy policy directly